package.json

```diff
@@ -1,41 +1,37 @@
 {
```

Comment:
oh I see.  Well when a script gets too big, I usually move it out into its own file.